extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -129,18 +132,43 @@ export class AmpStoryReaction extends AMP.BaseElement {
     this.variableService_ = getVariableService(this.win);
   }
 
+  /**
+   * Gets the root element.
+   * @visibleForTesting
+   * @return {?Element}
+   */
+  getRootElement() {
+    return this.rootEl_;
+  }
+
+  /**
+   * Gets the options.
+   * @visibleForTesting
+   * @return {!Array<!Element>}
+   */
+  getOptionElements() {
+    return toArray(
```

Comment:
Why did you remove the property that was caching these?

Reply:
That was the original PR, but I'll add it here since it's not a big change